CHANGES.rst

```diff
@@ -1,6 +1,20 @@
 Changes
 =======
 
+3.4b1 (2019-08-17)
```

Comment:
```suggestion
3.4 (TBA)
```
I'd avoid release notes for 3.4b1, and go straight into 3.4; 3.4b1 is not supposed to be a public release anyways.